README.md

```diff
@@ -278,6 +278,59 @@ AnthropicClient client = AnthropicOkHttpClient.builder()
     .build();
 ```
 
+### Streaming Helpers
+
+The SDK provides conveniences for streamed messages. A `MessageAccumulator` can record the stream
+of events in the response as they are processed and accumulate a `Message` object similar to that
+which would have been returned by the non-streaming API.
+
+A `BetaMessageAccumulator` is also available for the accumulation of a `BetaMessage` object. It is
+used in the same manner as the `MessageAccumulator`.
+
+For a synchronous response add a `Stream.peek()` call to the stream pipeline to accumulate each 
```

Comment:
Let's link the inline code to the method. Like here: https://github.com/anthropics/anthropic-sdk-java/blob/71418d53ad4b1d7547f1280953e33bee91a97c35/README.md?plain=1#L250C66-L250C70